static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -39,8 +55,10 @@ $ tree .
 ├── stages
 │   └── test.dvc
 └── test.txt
+```
 
-
-If you check your file using `git mv` then it shows that your path isn't
-updated but the DVC files and/or directories are relocated to destination. So,
-using `git mv` is not an option.
+If you change your file using `mv source destination` then it shows that your
```

Comment:
I still don't understand how is it relevant to DVC tbh

Reply:
I have explained it a bit more.